### DISCUSSION TITLE
how do i bind a keypress to copy current line diagnostic?

### DISCUSSION THREAD

**Discussion by lsshawn** [LazyVim/LazyVim#693] [Q&A]
![image](https://user-images.githubusercontent.com/28279366/235479249-617cbe2f-5fcc-43f3-b90e-3cd80d4c101c.png)

**Comment by dpetka2001** [comment]
I don't know how to outright do what you asked. But as a workaround you could open Telescope diagnostics with `<leader>sd` and then `C-q` on the entry you want to send it to quickfixlist. When on quickfixlist you can yank whatever you like. A bit more key presses but couldn't think of anything else.
